### PR TITLE
[DROP-IN] Overriding overview pitch to 0.0 when in Free Drive.

### PR DESCRIPTION
**Pull request by tomaszrybakiewicz**
Closes [#1533](https://github.com/mapbox/navigation-sdks/issues/1533)

### Screenshots or Gifs

_QA App Screen recording on Pixel 2_

https://user-images.githubusercontent.com/2678039/161787748-c4e476d7-1942-43fa-a4da-ab75a7483d6a.mp4

### CODE REVIEW

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/camera/CameraComponent.kt [hunk 1]
```diff
@@ -89,6 +89,7 @@ internal class CameraComponent constructor(
                     is NavigationState.FreeDrive -> {
                         viewportDataSource.followingZoomPropertyOverride(FOLLOWING_ZOOM_OVERRIDE)
                         viewportDataSource.overviewZoomPropertyOverride(OVERVIEW_ZOOM_OVERRIDE)
+                        viewportDataSource.overviewPitchPropertyOverride(OVERVIEW_PITCH_OVERRIDE)
```
Comment (abhishek1508) [inline libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/camera/CameraComponent.kt:92]: Can we add some unit tests?

**Review by abhishek1508 — COMMENTED** [review]
LGTM. Left a comment

**Comment by RingerJK** [comment]
@abhishek1508 as far as you know we divided tests on UI and Core and jacoco results are uploaded independently to codecov. So codecov updated them independently and merge them. You can have a look at the original https://github.com/mapbox/mapbox-navigation-android/pull/5658#issuecomment-1088933129 it already contains relevant values. 

I tried to solve this issue with `after_n_builds: 2` settings, but looks like it's not working  
https://github.com/mapbox/mapbox-navigation-android/blob/697bf68df730b86644583bb144ea3940b470d24b/codecov.yml#L14

Another option is to wait for all CI checks and then push statistics to a pull request. Suppose, make sense to enable it to avoid data confusion

**Comment by RingerJK** [comment]
>Cool. Thanks for the insight. Could you cut a ticket for this and take a look at it when you get a chance?

As far as I got from [docs](https://docs.codecov.com/docs/pull-request-comments#after_n_builds) `after_n_builds` must work. Let's give a few rounds of codecov reports and check how it works. It might be a one-time bug or something like that

File: libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/camera/CameraComponent.kt (hunk 1 above)
Comment (tomaszrybakiewicz) [reply libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/camera/CameraComponent.kt:92]: Done

**Review by abhishek1508 — APPROVED** [review]
Thanks for addressing the feedback!

**Pull request merged**